pallets/messages/src/lib.rs

```diff
@@ -179,16 +191,15 @@ pub mod pallet {
 			schema_id: SchemaId,
 			/// The block number for these messages
 			block_number: T::BlockNumber,
-			/// Number of messages in this block for this schema
```

Comment:
This might cause some backwards compatibility issues for Peter, but I think it will worth it due to simplifying a lot of event related problems

Reply:
I do not understand the reasoning we are leaving it. It feels like we are worrying about a problem we do not have.

Reply:
True we have a limited number of integrations, but setting I want to set the patterns across the board.

This event in particular is being used by at least Peter. Let's keep the churn low as possible.